uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/PublicHolidayLeaveRequestCreationTest.php

```diff
@@ -64,4 +67,38 @@ public function testItUpdatesTheBalanceChangeForOverlappingLeaveRequestDayToZero
 
     $this->assertEquals(0, LeaveBalanceChange::getTotalBalanceChangeForLeaveRequest($leaveRequest));
   }
+
+  public function testCanCreatePublicHolidayLeaveRequestsForAnAbsenceType() {
+    $contact = ContactFabricator::fabricate(['first_name' => 'Contact 1']);
```

Comment:
you can fabricate a contact without passing any data since you didn't use the passed parameters